{
  "affected_release": [
    {
      "product_name": "Red Hat Enterprise Linux 3",
      "release_date": "2005-02-10T00:00:00Z",
      "advisory": "RHSA-2005:136",
      "package": "mailman-3:2.1.5-24.rhel3",
      "cpe": "cpe:/o:redhat:enterprise_linux:3"
    },
    {
      "product_name": "Red Hat Enterprise Linux 4",
      "release_date": "2005-02-15T00:00:00Z",
      "advisory": "RHSA-2005:137",
      "package": "mailman-3:2.1.5-31.rhel4",
      "cpe": "cpe:/o:redhat:enterprise_linux:4"
    }
  ],
  "package_state": null,
  "threat_severity": "Important",
  "public_date": "2005-02-09T00:00:00Z",
  "bugzilla": {
    "description": "CVE-2005-0202 security flaw",
    "id": "1617490",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1617490"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "",
    "cvss3_scoring_vector": "",
    "status": ""
  },
  "iava": "",
  "cwe": "",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2005-0202",
  "document_distribution": "",
  "details": [
    "Directory traversal vulnerability in the true_path function in private.py for Mailman 2.1.5 and earlier allows remote attackers to read arbitrary files via \".../....///\" sequences, which are not properly cleansed by regular expressions that are intended to remove \"../\" and \"./\" sequences."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2005-0202\nhttps://nvd.nist.gov/vuln/detail/CVE-2005-0202"
  ]
}